.github/bors.toml

```diff
@@ -1,2 +1,2 @@
 delete_merged_branches = true
-status = ["ci"]
\ No newline at end of file
+status = ["test"]
```

Comment:
```suggestion
status = [
    "test (stable, ubuntu-latest)",
    "test (stable, macOS-latest)",
    "test (stable, windows-latest)",
    "test (nightly, ubuntu-latest)",
    "test (nightly, macOS-latest)",
    "test (nightly, windows-latest)",
]
```

Reply:
would be nice to not need to list all the matrix combinations here. I'll add a note to the `ci.yml` stating that this other file needs to be updated if the matrix is changed (or a new job is added).